web/ui/templates/alerts.html

```diff
@@ -38,6 +43,21 @@ <h2>Alerts</h2>
               <td>{{.ActiveAt.UTC}}</td>
               <td>{{.Value}}</td>
             </tr>
+                {{ if .Annotations.Map}}
+                  <tr style="display:none" class="alert_annotations">
```

Comment:
The <tr should be aligned with the others

Reply:
I have aligned the code